readme.md

```diff
@@ -0,0 +1,30 @@
+![Polylang](assets/polylang-logo.png)
+
+Welcome to the Polylang repository on GitHub. Here you can browse the source, look at open
+issues and keep track of development.
+
+If you are not a developer, please use the [Polylang plugin page](https://wordpress.org/plugins/polylang/) on [WordPress.org](https://wordpress.org).
```

Comment:
I suggest: 
`we recommend to [download Polylang](https://wordpress.org/plugins/polylang/) from the WordPress directory.`
instead of
`please use the [Polylang plugin page](https://wordpress.org/plugins/polylang/) on [WordPress.org](https://wordpress.org).`